benchmarks/src/main/java/com/google/samples/apps/nowinandroid/foryou/ForYouActions.kt

```diff
@@ -22,6 +22,11 @@ import androidx.test.uiautomator.Until
 import androidx.test.uiautomator.untilHasChildren
 import com.google.samples.apps.nowinandroid.flingElementDownUp
 
+fun MacrobenchmarkScope.allowNotifications() {
+    val dialog = device.wait(Until.findObject(By.text("Allow")), 5_000)
+    dialog?.click()
```

Comment:
Am I missing a device.waitForIdle() ?

Reply:
Tested on an emulator with API 33 and this works just fine 👍